src/main.zig

```diff
@@ -2800,6 +2800,12 @@ pub fn cmdInit(
         error.FileNotFound => {},
         else => fatal("unable to test existence of build.zig: {s}\n", .{@errorName(err)}),
     }
+    if (fs.cwd().access("src" ++ s ++ "main.zig", .{})) |_| {
+        fatal("existing src" ++ s ++ "main.zig file would be overwritten", .{});
+    } else |err| switch (err) {
+        error.FileNotFound => {},
+        else => fatal("unable to test existence of src" ++ s ++ "main.zig: {s}\n", .{@errorName(err)}),
```

Comment:
IMO, the better thing to do here would just be to overwrite. This should not be a hard error.

Reply:
Why not? If the file system is returning an error on this operation then it is kind of unlikely that writing will succeed anyways. Better to play it safe and fail early than risk overwriting someone's existing main.zig.